tpm/tpm_test.go

```diff
@@ -70,6 +70,28 @@ func TestGetManufacturer(t *testing.T) {
 	t.Logf("TPM VendorID: %v\n", vendorID)
 }
 
+func TestGetAlgs(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+
+	algs, err := GetAlgs(rwc)
+	if err != nil {
+		t.Fatal("Couldn't read Algorithms")
+	}
+	t.Logf("Algorithms are: %v\n", algs)
```

Comment:
can you compare this against a known set of good values?
the spec says "The TPM MUST support the algorithms TPM_ALG_RSA, TPM_ALG_SHA, TPM_ALG_HMAC, and TPM_ALG_MGF1"